ansible_collections/arista/avd/roles/eos_designs/doc/management-settings.md

```diff
@@ -78,7 +78,14 @@ snmp_settings:
   # Generate a local engineId for SNMP by hashing via SHA1 the string
   # generated via the concatenation of the hostname plus the management IP.
   # {{ inventory_hostname }} + {{ switch.mgmt_ip }}
+  # This SHOULD NOT be used with `default_engineid_from_system_mac`
```

Comment:
Could we simplify this by creating a `compute_local_engineid_source: < "hostname_and_ip" | "system_mac" | default -> "hostname_and_ip" >`
Then we don't have "competing" vars, but instead offer customization and future enhancements.

Reply:
It would also reduce the amount of doc for other vars we would need to update. Ex. check line 89.

Reply:
there should not be a default value but yeah we can do this - by default it should stay disabled I think

Let me rework it with your suggestion!

Reply:
ok following our chat - yes new variable with a default_value ;)